### PR TITLE
Update FUNDING.md to add core maintainer stipend

### PR DESCRIPTION
**Pull request by FredKSchott**
looking for feedback! Inspired by eslint

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
I love this — thanks so much for kicking it off Fred.

One thing coming out of the last round of community awards might be some guidance on what to submit to OC when claiming this? Link to show core maintainer status? Summary of cool (or not cool but very helpful!) stuff you worked on? 

Guidance doesn't have to be here necessarily — could also be in a Notion doc core maintainers have access to or something.

This text reads well to me though. Excited about this initiative!

File: FUNDING.md
Comment (Yan-Thomas) [inline FUNDING.md:47]: I think "per hour" doesn't have hyphen!
```suggestion
Right now, the stipend is calculated per hour at $50/hour, with a maximum of $1000 each month. These numbers may shift as our funding and team size change over time.
```

Comment (Yan-Thomas) [inline FUNDING.md:47]: According to my reality, the stipend value per hour is great, being even more than what most of my family makes monthly, for comparison.

In my opinion, the way the worked hours are counted should be very clear here. This is important **both** to the Astro community, as in to know the folks being benefited are in fact helping, and to also protect the core maintainer from being wrongfully accused of somehow "stealing" it.

I can say I spend 3 to 4 hours working in Astro every day, but some of my most time-consuming contributions aren't easy to display, like accessibility research that ends up resulting in 10 lines of HTML/CSS changed, which someone might claim it's just "10 minutes of work".

**Review by Yan-Thomas — COMMENTED** [review]
I added a few technical comments and writing suggestions here. Though, as I'm one of the folks that would be benefited by this, I abstain from agreeing/disagreeing on the initiative.

File: FUNDING.md
Comment (sarah11918) [reply FUNDING.md]: It might not be as important to be as explicit here about how/tracking of hours as it might be to say that hours may be "subject to validation." This leaves our options open, and allows us to make judgement calls.

For example, active core users who are engaged and contributing regularly can make a claim for 20 hours and I don't think anyone feels the need to validate/verify that. If a contributor that no one has seen in a while makes a large claim, that's going to look strange, and just a simple "subject to verification" gives us enough cover to ask questions and validate before accepting.

I think it's also worth noting that the amount of trust someone would have earned by reaching core is pretty high. And, the nature of contributions as a community core member can look wildly different. Ultimately, it's about whether someone has been providing value to our community. I'd argue that it might even be easier to be specific about when someone *hasn't* been, than when they have... because our core members provide so much value just by being actively around!

**Comment by sarah11918** [comment]
Heartily approve, and left a comment on Yan's comment with a suggestion that we could tack in somewhere a little "subject to validation" note if desired to provide an extra level of cover in the event of a suspicious claim.

File: FUNDING.md
Comment (FredKSchott) [reply FUNDING.md]: Going through Open Collective force you to include an invoice, so I think we can leave it up to every contributor to leave a detailed invoice. Maybe not minute-by-minute, but measured in terms of hours and fractions of hours would work.

**Review by natemoo-re — APPROVED** [review]
Another hearty approval—I love this initiative!

I could see us expanding this program to maintainers as well, but let's see how this goes first. 👍

**Review by Jutanium — APPROVED** [review]
I'm so glad we're adding this! I might change "participating in the Discord server" to a more specific Discord example like "answering support questions".

**Comment by sarah11918** [comment]
Maybe does not have to be written into this document, but as this is brand spanking new and we have some eligible contributors, it might be worth at least thinking about whether this is "calendar month", whether there is a "day/deadline" for invoicing for the previous month etc.

**Pull request merged**
